content/resources/radio.md

```diff
@@ -5,55 +5,71 @@ menu:
     main:
         parent: resources
 ---
+<style>
+.sstv img {
+	width: 32%;
+}
 
-# History
+</style>
+{{% picture "radio/FT991.jpg" %}}
 
-We did have a Amateur Radio Society club which was run by the students which was
-very successful through the 70s and 80s up to about 1995.
 
-It's popularity declined after 1995 and then it was very difficult to get the 15
-students to sign up, that the student union required to form the club.
+We also run a radio workshop every few weekends, allowing members to work on obtaining their amateur radio licenses, and try and make contact with people across Europe with their callsigns! We currently have more than 10 members who have passed their Foundation exams and more than 5 have Intermediate Licenses. 

```

Comment:
HaHa fair enough, suppose it's not too bad if they plan to run something this year 👍